### ISSUE TITLE
Bundled translations maintenance status

### ISSUE THREAD

**Issue by rob006** [v17development/flarum-seo#61]
What is the status of translations provided in https://github.com/v17development/flarum-seo/tree/master/locale? I'm trying to cleanup redundant translations from [Weblate](https://weblate.rob006.net/projects/flarum/) - if translation for given language is already bundled with extension, there is no sense to have it duplicated in Weblate. But this is true only for actively maintained translations, which usually means only translations maintained by extension author. For example if extension author knows Italian, he can provide `en` and `it` translations, while the rest should probably be handled inside of respective language packs. So now I'm trying to figure out which translations are available because of extension author knows this language and will be able to maintain them, and which are just leftovers from old days where translations were contributed by community to extensions instead of language packs (in that case translations may become outdated after some times, since nobody will update them).

So how does it looks like for https://github.com/v17development/flarum-seo/tree/master/locale? Which languages do you know and will be able to update if any change will be needed?

**Comment by jaspervriends** [comment]
Hi Rob, thank you for opening this issue. Currently we would only be able to provide updates for Dutch, English, Frisian and German languages. The other onese we don't speak.

Since Weblate is starting to becoming more usual for Flarum, we could remove the translations of the languages in this repository that are already in weblate, like the Dutch, French, and Spanish language.

Let me know how you'd like to proceed.

**Comment by rob006** [comment]
Well, that's up to you. Personally I think that you should keep Dutch translation in your extension, as this is in general more reliable way of handling translations. The only reason why we keep translations in language packs is that it is really problematic for extension author to verify translations in unknown language. But this is not the case if you know this language - you can update translation by yourself and verify suggestions from community. So unless it is problematic for you to maintain Dutch translation, I would leave it in extension and remove from Weblate.

As for others: I will keep them available in Weblate, and maybe try to copy missing translations from this extensions, so language packs should have them covered. That means that you could remove them from extension, but I suggest to wait with that for major release, as this may be treated as BC break.

**Comment by rob006** [comment]
I removed Dutch language from Weblate to not duplicate work. But if you decide to remove it from extension and rely on language pack, then let me know. 

As for French and Spanish: both are now translated on Weblate (I copied translation for Spanish, French was already translated), so it should be safe to remove these translations from extension in the future.

As for others - I don't track translations outside of Weblate, so I can't say much about them. But if new language will be added to Weblate, I'll let you know, so you will know which translations can be safely removed from this extension.
